### ISSUE TITLE
🐞 Bug report: Containers locked

### ISSUE THREAD

**Issue by Pixcil** [elementor/elementor#19373]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

When I attempt to edit an existing page with containers, containers are locked and a (/) icon is presented. Sections working as expected. See Gyazo recording: https://gyazo.com/be0a2ad48aadd9226ee1efed8698f32b  (Top half is section, bottom half is containers)

As per recording, I can insert new widgets in the Sections, but not in the containers. If I create a new container, this can be edited while the page to edit is still open. If I save and exit, come back, they are also locked. Existing containers and widgets can be edited and deleted but cannot have other widgets inserted, even if placed outside of the container first.

This happens with Edge and Chrome. Tested PHP 7.4, 8.0, 8.1. 
Tested with all plugins disabled and a blank '2022' theme, only Elementor and Elementor Pro active.
Tested complete website and DB wipe and fresh install, issue still present.

### Steps to reproduce

Create a page with Containers.
Save the page and close it.
Go back in to edit the page.
Attempt to add widgets to containers.

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently using the steps above.

### System Info

== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: (Ubuntu) v8.0.28
	PHP Version: 8.1.0
	PHP Memory Limit: 128M
	PHP Max Input Vars: 1000
	PHP Max Post Size: 512M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.0.1
	Site URL: https://www.eigo81.com
	Home URL: https://www.eigo81.com
	WP Multisite: No
	Max Upload Size: 512 MB
	Memory limit: 40M
	Max Memory limit: 256M
	Permalink Structure: /%postname%/
	Language: en-US
	Timezone: 9
	Admin Email: admin@eigo81.com
	Debug Mode: Inactive

== Theme ==
	Name: BuddyBoss Child
	Version: 1.0.1
	Author: BuddyBoss.com
	Child Theme: Yes
	Parent Theme Name: BuddyBoss Theme
	Parent Theme Version: 2.0.7
	Parent Theme Author: BuddyBoss

== User ==
	Role: administrator
	WP Profile lang: en_US
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/104.0.0.0 Safari/537.36

== Active Plugins ==
	AyeCode Connect
		Version: 1.2.11
		Author: AyeCode

	BdThemes Testimonials
		Version: 1.0.0
		Author: BdThemes

	BuddyBoss Platform
		Version: 2.0.7
		Author: BuddyBoss

	BuddyBoss Platform Pro
		Version: 2.0.5
		Author: BuddyBoss

	BuddyPress Profanity Filter
		Version: 1.1.2
		Author: BuddyDev

	BuddyPress Profile Data Control
		Version: 1.0.1
		Author: BuddyDev

	BuddyPress Profile Visibility Manager
		Version: 1.9.2
		Author: BuddyDev

	BuddyPress User Badges
		Version: 1.2.4
		Author: BuddyDev

	BuddyPress User Profile Tabs Creator Pro
		Version: 1.3.0
		Author: BuddyDev

	Crocoblock Wizard
		Version: 1.2.4
		Author: Crocoblock

	CSS Hero
		Version: 5.0.5
		Author: The CSS Hero Team

	DreamHost Panel Login
		Version: 1.0.0
		Author: DreamHost, danielbachhuber

	Elementor
		Version: 3.6.8
		Author: Elementor.com

	Elementor Pro
		Version: 3.7.3
		Author: Elementor.com

	Element Pack Pro
		Version: 6.4.1
		Author: BdThemes

	Eventin Pro
		Version: 3.2.2
		Author: Themewinter

	Facebook Like User Activity Stream For BuddyPress
		Version: 1.3.2
		Author: Brajesh Singh

	GetPaid
		Version: 2.6.14
		Author: AyeCode Ltd

	GetPaid Advertising
		Version: 1.0.8
		Author: AyeCode Ltd

	GetPaid Stripe Payments
		Version: 2.2.10
		Author: AyeCode Ltd

	JetBlocks For Elementor
		Version: 1.3.2
		Author: Crocoblock

	JetBlog For Elementor
		Version: 2.3.1
		Author: Crocoblock

	JetElements For Elementor
		Version: 2.6.6
		Author: Crocoblock

	JetEngine
		Version: 3.0.2
		Author: Crocoblock

	JetEngine - Custom visibility conditions
		Version: 1.1.1
		Author: Crocoblock

	JetEngine - Data Stores - clear posts on delete
		Version: 1.0.0
		Author: Crocoblock

	JetEngine - fill empty space in the Listing grid
		Version: 1.0.0
		Author: Crocoblock

	JetEngine - Get value from repeater field
		Version: 1.0.0
		Author: Crocoblock

	JetEngine - Listing’s counter
		Version: 1.0.0
		Author: Crocoblock

	JetEngine Forms - popup notification
		Version: 1.1.2
		Author: Crocoblock

	JetFormBuilder
		Version: 2.1.2
		Author: Crocoblock

	JetFormBuilder Hierarchical Select
		Version: 1.0.1
		Author: Crocoblock

	JetFormBuilder Limit Form Responses
		Version: 1.0.3
		Author: Crocoblock

	JetFormBuilder Stripe Gateway
		Version: 1.1.0
		Author: Crocoblock

	Jet Hide Empty Items
		Version: 1.0.0
		Author: Crocoblock

	JetMenu
		Version: 2.2.2
		Author: Crocoblock

	JetPlugins Dynamic Data Addon
		Version: 1.3.3
		Author: Crocoblock

	JetPopup
		Version: 1.5.6
		Author: Crocoblock

	JetReviews For Elementor
		Version: 2.2.7
		Author: Crocoblock

	JetSearch
		Version: 3.0.1
		Author: Crocoblock

	JetSmartFilters
		Version: 2.3.13
		Author: Crocoblock

	JetSmartFilters - Advanced date filter
		Version: 1.0.0
		Author: Crocoblock

	JetTabs For Elementor
		Version: 2.1.18
		Author: Crocoblock

	Meta Box
		Version: 5.6.6
		Author: MetaBox.io

	Newsletter
		Version: 7.5.1
		Author: Stefano Lissa & The Newsletter Team

	Newsletter - Addons Manager
		Version: 1.1.8
		Author: The Newsletter Team

	Newsletter - Amazon SES
		Version: 1.3.8
		Author: The Newsletter Team

	Newsletter - BuddyPress Integration
		Version: 1.0.5
		Author: The Newsletter Team

	Newsletter - Elementor Addon
		Version: 1.0.4
		Author: The Newsletter Team

	Newsletter - Extended Composer Blocks
		Version: 1.3.8
		Author: The Newsletter Team

	Newsletter - Reports and Retargeting
		Version: 4.4.5
		Author: The Newsletter Team

	WP Easy Updates
		Version: 1.1.18
		Author: AyeCode Ltd

	WP Event Solution
		Version: 3.2.3
		Author: Themewinter

== Elements Usage ==
	
	wp-page : 3
		container : 7
		form : 1
		heading : 2
		image : 3
		text-editor : 4

== Elementor Experiments ==
	Optimized DOM Output: Active by default
	Improved Asset Loading: Active by default
	Improved CSS Loading: Active by default
	Inline Font Icons: Inactive by default
	Accessibility Improvements: Active by default
	Import Export Template Kit: Active by default
	Additional Custom Breakpoints: Active by default
	Hide native WordPress widgets from search results: Active by default
	: Inactive by default
	Flexbox Container: Active
	Default to New Theme Builder: Active by default
	Landing Pages: Active by default
	Color Sampler: Active by default
	Favorite Widgets: Active by default
	Admin Top Bar: Active by default
	Page Transitions: Active by default
	Notes: Active by default
	Form Submissions: Active
	Scroll Snap: Active by default

== Log ==
	
PHP: showing 12 of 12PHP: 2022-08-06 07:52:59 [warning X 1][/home/eigo81/eigo81.com/wp-content/plugins/elementor/core/common/modules/connect/module.php::230] Attempt to read property &quot;email&quot; on null [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2022-08-06 07:56:24 [notice X 600][/home/eigo81/eigo81.com/wp-content/plugins/elementor/modules/dev-tools/deprecation.php::301] The $finder_category_name argument is deprecated since version 3.5.0! [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2022-08-06 08:03:40 [notice X 493][/home/eigo81/eigo81.com/wp-content/plugins/elementor/modules/dev-tools/deprecation.php::301] The $control_id argument is deprecated since version 3.5.0! [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2022-08-06 19:19:26 [notice X 10][/home/eigo81/eigo81.com/wp-content/plugins/elementor/modules/dev-tools/deprecation.php::301] The $finder_category_name argument is deprecated since version 3.5.0! [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> rest_error_handler()
#1: /home/eigo81/eigo81.com/wp-content/plugins/elementor/modules/dev-tools/deprecation.php(301): class type trigger_error()
#2: /home/eigo81/eigo81.com/wp-content/plugins/elementor/core/common/modules/finder/categories-manager.php(68): Elementor\Modules\DevTools\Deprecation -> deprecated_argument()
#3: /home/eigo81/eigo81.com/wp-content/plugins/elementor/core/common/modules/finder/categories-manager.php(50): Elementor\Core\Common\Modules\Finder\Categories_Manager -> register()
#4: /home/eigo81/eigo81.com/wp-content/plugins/jet-popup/includes/elementor-finder/elementor-finder.php(47): Elementor\Core\Common\Modules\Finder\Categories_Manager -> add_category()
',
)]
PHP: 2022-08-06 19:20:05 [notice X 17][/home/eigo81/eigo81.com/wp-content/plugins/elementor/modules/dev-tools/deprecation.php::301] The $control_id argument is deprecated since version 3.5.0! [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> rest_error_handler()
#1: /home/eigo81/eigo81.com/wp-content/plugins/elementor/modules/dev-tools/deprecation.php(301): class type trigger_error()
#2: /home/eigo81/eigo81.com/wp-content/plugins/elementor/includes/managers/controls.php(523): Elementor\Modules\DevTools\Deprecation -> deprecated_argument()
#3: /home/eigo81/eigo81.com/wp-content/plugins/elementor/includes/managers/controls.php(501): Elementor\Controls_Manager -> register()
#4: /home/eigo81/eigo81.com/wp-content/plugins/bdthemes-element-pack/includes/controls/select-input/dynamic-select.php(124): Elementor\Controls_Manager -> register_control()
',
)]
PHP: 2022-08-06 19:20:05 [notice X 17][/home/eigo81/eigo81.com/wp-content/plugins/elementor/modules/dev-tools/deprecation.php::301] The $control_id argument is deprecated since version 3.5.0! [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> rest_error_handler()
#1: /home/eigo81/eigo81.com/wp-content/plugins/elementor/modules/dev-tools/deprecation.php(301): class type trigger_error()
#2: /home/eigo81/eigo81.com/wp-content/plugins/elementor/includes/managers/controls.php(523): Elementor\Modules\DevTools\Deprecation -> deprecated_argument()
#3: /home/eigo81/eigo81.com/wp-content/plugins/elementor/includes/managers/controls.php(501): Elementor\Controls_Manager -> register()
#4: /home/eigo81/eigo81.com/wp-content/plugins/bdthemes-element-pack/loader.php(698): Elementor\Controls_Manager -> register_control()
',
)]
PHP: 2022-08-06 19:20:05 [notice X 17][/home/eigo81/eigo81.com/wp-content/plugins/elementor/modules/dev-tools/deprecation.php::301] The $control_id argument is deprecated since version 3.5.0! [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> rest_error_handler()
#1: /home/eigo81/eigo81.com/wp-content/plugins/elementor/modules/dev-tools/deprecation.php(301): class type trigger_error()
#2: /home/eigo81/eigo81.com/wp-content/plugins/elementor/includes/managers/controls.php(523): Elementor\Modules\DevTools\Deprecation -> deprecated_argument()
#3: /home/eigo81/eigo81.com/wp-content/plugins/elementor/includes/managers/controls.php(501): Elementor\Controls_Manager -> register()
#4: /home/eigo81/eigo81.com/wp-content/plugins/bdthemes-element-pack/loader.php(706): Elementor\Controls_Manager -> register_control()
',
)]
PHP: 2022-08-06 19:20:05 [notice X 17][/home/eigo81/eigo81.com/wp-content/plugins/elementor/modules/dev-tools/deprecation.php::301] The $control_id argument is deprecated since version 3.5.0! [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> rest_error_handler()
#1: /home/eigo81/eigo81.com/wp-content/plugins/elementor/modules/dev-tools/deprecation.php(301): class type trigger_error()
#2: /home/eigo81/eigo81.com/wp-content/plugins/elementor/includes/managers/controls.php(523): Elementor\Modules\DevTools\Deprecation -> deprecated_argument()
#3: /home/eigo81/eigo81.com/wp-content/plugins/elementor/includes/managers/controls.php(501): Elementor\Controls_Manager -> register()
#4: /home/eigo81/eigo81.com/wp-content/plugins/jet-popup/includes/integration.php(134): Elementor\Controls_Manager -> register_control()
',
)]
PHP: 2022-08-06 20:24:02 [error X 1][/home/eigo81/eigo81.com/wp-content/plugins/elementor-pro/modules/assets-manager/asset-types/fonts/typekit-fonts.php::240] Uncaught TypeError: count(): Argument #1 ($value) must be of type Countable|array, bool given in /home/eigo81/eigo81.com/wp-content/plugins/elementor-pro/modules/assets-manager/asset-types/fonts/typekit-fonts.php:240
Stack trace:
#0 /home/eigo81/eigo81.com/wp-content/plugins/elementor-pro/modules/assets-manager/asset-types/fonts/typekit-fonts.php(240): count(false)
#1 /home/eigo81/eigo81.com/wp-includes/class-wp-hook.php(307): ElementorPro\Modules\AssetsManager\AssetTypes\Fonts\Typekit_Fonts-&gt;integrations_admin_ajax_handler(&#039;&#039;)
#2 /home/eigo81/eigo81.com/wp-includes/class-wp-hook.php(331): WP_Hook-&gt;apply_filters(&#039;&#039;, Array)
#3 /home/eigo81/eigo81.com/wp-includes/plugin.php(476): WP_Hook-&gt;do_action(Array)
#4 /home/eigo81/eigo81.com/wp-admin/admin-ajax.php(187): do_action(&#039;wp_ajax_element...&#039;)
#5 {main}
  thrown [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2022-08-08 06:39:55 [warning X 1][/home/eigo81/eigo81.com/wp-content/plugins/elementor/core/common/modules/ajax/module.php::171] Undefined array key &quot;data&quot; [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2022-08-08 09:20:21 [notice X 5][/home/eigo81/eigo81.com/wp-content/plugins/elementor-pro/core/database/model-base.php::135] Return type of ElementorPro\Core\Database\Model_Base::jsonSerialize() should either be compatible with JsonSerializable::jsonSerialize(): mixed, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice [array (
  'trace' => '
#0: /home/eigo81/eigo81.com/wp-content/plugins/elementor-pro/core/database/model-base.php(10): Elementor\Core\Logger\Manager -> rest_error_handler()
#1: /home/eigo81/eigo81.com/wp-content/plugins/elementor-pro/plugin.php(153): class type include()array (
  0 => '/home/eigo81/eigo81.com/wp-content/plugins/elementor-pro/core/database/model-base.php',
)
#2: /home/eigo81/eigo81.com/wp-content/plugins/elementor-pro/modules/notes/database/models/note.php(18): ElementorPro\Plugin -> autoload()
#3: /home/eigo81/eigo81.com/wp-content/plugins/elementor-pro/plugin.php(153): class type include()array (
  0 => '/home/eigo81/eigo81.com/wp-content/plugins/elementor-pro/modules/notes/database/models/note.php',
)
#4: /home/eigo81/eigo81.com/wp-content/plugins/elementor-pro/modules/notes/data/controller.php(236): ElementorPro\Plugin -> autoload()
',
)]
PHP: 2022-08-08 09:21:12 [notice X 2][/home/eigo81/eigo81.com/wp-content/plugins/elementor-pro/core/database/model-base.php::135] Return type of ElementorPro\Core\Database\Model_Base::jsonSerialize() should either be compatible with JsonSerializable::jsonSerialize(): mixed, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

Log: showing 7 of 72022-08-06 07:57:35 [info] elementor-pro::elementor_pro_updater Started 
2022-08-06 07:57:35 [info] Elementor Pro/Upgrades - _on_each_version Start  
2022-08-06 07:57:35 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2022-08-06 07:57:35 [info] Elementor Pro/Upgrades - _v_3_7_2_woocommerce_rename_related_to_related_products Start  
2022-08-06 07:57:35 [info] Elementor Pro/Upgrades - _v_3_7_2_woocommerce_rename_related_to_related_products Finished 
2022-08-06 07:57:35 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.7.1',
  'to' => '3.7.3',
)]
2022-08-06 07:57:35 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.7.1',
  'to' => '3.7.3',
)]

JS: showing 1 of 1JS: 2022-08-07 21:18:21 [error X 36][https://www.eigo81.com/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.6.8:3:459034] Cannot read properties of undefined (reading \&#039;parent\&#039;) 

== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatibility not specified
	JetBlocks For Elementor: Compatibility not specified
	JetBlog For Elementor: Compatibility not specified
	JetElements For Elementor: Compatibility not specified
	JetReviews For Elementor: Compatibility not specified
	JetTabs For Elementor: Compatibility not specified
	Newsletter - Elementor Addon: Compatibility not specified

== Elementor Pro - Compatibility Tag ==

**Comment by GALASIO** [comment]
+1

**Comment by simkoG** [comment]
Same error. JS console shows an error when this happens: 

```
editor.min.js?ver=3.6.7:3 Uncaught TypeError: Cannot read properties of undefined (reading 'parent')
    at n.getNestingLevel (editor.min.js?ver=3.6.7:3:475108)
    at editor.min.js?ver=3.6.7:3:478411
```

I have disabled all plugins (and mu-plugins) and set the theme to the default one.

![image](https://user-images.githubusercontent.com/10430702/183606445-c81f05d9-f85b-44c6-a304-31c09963a830.png)

**Comment by Pixcil** [comment]
This has fixed for me after the recent update and containers are now behaving as expected.

**Comment by simkoG** [comment]
@Pixcil confirmed. Fixed in Elementor (free) 3.7.0.
